.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
remove this?

Reply:
it is added by defalut when using CLion, so add this is better.